hack/init-environment.sh

```diff
@@ -156,6 +156,57 @@ function install_flux () {
     --set='rbac.pspEnabled=true'
 }
 
+#####
+## Set up and install the Vault secret manager in the command center GKE cluster.
+#####
+function install_secret_manager () {
+  local -r kubeconfig=$1 env_dir=$2
+
+  # Install the CRD definitions separately. Helm doesn't have
+  # a coherent story for handling these yet (since updating them
+  # the wrong way can result in all existing objects being deleted),
+  # so they're easier to handle out-of-band.
+  docker run --rm -it \
+    -v ${kubeconfig}:/root/.kube/config:ro \
+    -v ${HOME}/.config:/root/.config:ro \
+    ${KUBECTL} \
+    # kubectl applys the secret-manager’s CRDs
+    apply -f \
+    https://raw.githubusercontent.com/tuenti/secrets-manager/master/config/crd/bases/secrets-manager.tuenti.io_secretdefinitions.yaml
```

Comment:
Let's pin a version instead of pointing at `master`, to avoid accidental problems.
```suggestion
    https://raw.githubusercontent.com/tuenti/secrets-manager/${SECRET_MANAGER_VERSION}/config/crd/bases/secrets-manager.tuenti.io_secretdefinitions.yaml
```
And:
```bash
declare -r SECRET_MANAGER_VERSION=v1.02
```
at the top of the script